### ISSUE TITLE
Requirements for building documentation inside docs_requirements.txt?

### ISSUE THREAD

**Issue by LeMeteore** [webmaven/morepath-tutorial#5]
Is it necessary to have a separate file for the packages that will be used to build morepath documentation?

**Comment by webmaven** [comment]
No. Building the morepath docs shouldn't be part of this tutorial, nor part of the meta-documentation on building this tutorial either.
